### ISSUE TITLE
npm install error

### ISSUE THREAD

**Issue by bluemix** [bbc/unicode-bidirectional#28]
Hello dear, I have the following error while installation 
`npm install unicode-bidirectional --save`

```
npm ERR! Error while executing:
npm ERR! /usr/bin/git ls-remote -h -t ssh://git@github.com/jameslawson/unicode-bidiclass.git
npm ERR! 
npm ERR! Permission denied (publickey).
npm ERR! fatal: Could not read from remote repository.
npm ERR! 
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.
npm ERR! 
npm ERR! exited with error code: 128

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/bluemix/.npm/_logs/2017-08-05T02_55_06_459Z-debug.log
```

**Comment by jameslawson** [comment]
Hi, this problem has now been fixed **in v2.0.5**. Can you please try again.

The problem was partly caused by unicode-directional having an [SSH GitHub dependency](https://github.com/bbc/unicode-bidirectional/blob/2a421b362df576a9940a95bb565b0f963ca57079/package.json#L42) (`ssh://git@github.com/foo/bar.git`). This has now been removing (and replaced with a normal npm [semver version](https://github.com/bbc/unicode-bidirectional/blob/master/package.json#L42)).

**Comment by bluemix** [comment]
Ok, now it works correctly 👍 
thanks :)
